### PR TITLE
point new Rawhide to Fedora 40

### PR DESCRIPTION
**Pull request by lulinqing**

### CODE REVIEW

File: containers/provision/fedora:39.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+fedora:latest.sh
```
Comment (cbouchar) [inline containers/provision/fedora:39.sh:1]: This should have been a rename of fedora:37 to fedora:39.

**Comment by lulinqing** [comment]
Yeah I noticed. Renaming within Github.dev web IDE appears a bit weird.

Do you think that would work? If not I can get a local version from
terminal later.

**Comment by cbouchar** [comment]
I do all my operations from my pulled workspace.  I've never worked from web ui.

**Comment by lulinqing** [comment]
@cbouchar should be fixed now. The web IDE probably wasn't able to handle symlink well.

**Review by cbouchar — APPROVED** [review]
Sorry for delay.  I saw over the week-end and forgot to come back to it.

**Comment by lulinqing** [comment]
No problem, thanks!

**Review by StykMartin — APPROVED** [review]

**Pull request merged**